splunk/dasg_metrics_dashboard.xml

```diff
@@ -132,13 +100,32 @@
           <done>
             <set token="tokReturnBeneOkCount">$result.count$</set>
           </done>
-          <query>search "message.status"="OK" | search "message.fhir_id" > 0 | dedup message.fhir_id | table message.fhir_id | search NOT [search base=baseSearch1a | table message.fhir_id] | stats count</query>
+          <query>search "message.status"="OK" | search "message.fhir_id" &gt; 0 | dedup message.fhir_id | table message.fhir_id | search NOT [search base=baseSearch1a | table message.fhir_id] | stats count</query>
         </search>
         <option name="drilldown">all</option>
         <option name="height">226</option>
         <option name="refresh.display">progressbar</option>
       </single>
     </panel>
+    
+    <panel>
+      <title>1c. Beneficiaries Served (life to date cumulative)</title>
+      <search>
+          <init>
+            <set token="tokGroupTimestamp">$result.message.group_timestamp$</set>
+          </init>
+          <query>index=bluebutton source="/var/log/pyapps/perf_mon.log*" host=* env=impl | search "message.type"="global_state_metrics" | head 1 | table message.group_timestamp</query>
+      </search>
+      <single>
+        <search>
+          <query>index=bluebutton source="/var/log/pyapps/perf_mon.log*" host=* env=prod "message.type"="global_state_metrics" "message.group_timestamp"=$tokGroupTimestamp|s$ | table message.real_bene_cnt</query>
+        </search>
```

Comment:
@dtisza1 Thank you, all great comments! I was under the impression that this count would just be a constant reflection of the current total for all time. I will work on this some more to reflect the totals for the date ranges selected, that makes sense - I didn't consider that.

The "|s" I had to add so that the queries using that value would work, this adds double quotes to the value for the search queries. Resource for that info here: https://docs.splunk.com/Documentation/Splunk/8.2.1/Viz/tokens